docs/reference/installation/_interfaces/homebrew.cue

```diff
@@ -37,7 +37,7 @@ installation: _interfaces: homebrew: {
 				},
 				{
 					title:   "Configure Vector"
-					command: "fix me"
+					command: commands.configure
```

Comment:
I don't see this key in the `commands` map above. Is it somewhere else?

Reply:
@jszwedko it defaults to this:

https://github.com/timberio/vector/blob/ccd49073cb6d23ae0caf00b460411a95080a4827/docs/reference/installation.cue#L24-L30

Which is not very obvious. We could probably pull that up and make it explicit.

Reply:
Aha I see :+1: